scripts/netContain/scripts/contivNet.sh

```diff
@@ -114,34 +117,39 @@ if [ "$plugin" == "kubernetes" ]; then
    fi
 fi
 
+set +eo pipefail
+
 if [ $netmaster == true ]; then
 	echo "Starting netmaster "
 	while true; do
-		if [ "$cstore" != "" ]; then
-			/contiv/bin/netmaster $debug -cluster-mode $plugin -cluster-store $cstore -listen-url $listen_url -control-url $control_url &>/var/contiv/log/netmaster.log
-		else
-			/contiv/bin/netmaster $debug -cluster-mode $plugin -listen-url $listen_url -control-url $control_url &>/var/contiv/log/netmaster.log
+		if [ -f /tmp/restart_netmaster ]; then
+			if [ "$cstore" != "" ]; then
+				/contiv/bin/netmaster $debug -cluster-mode $plugin -cluster-store $cstore -listen-url $listen_url -control-url $control_url &>/var/contiv/log/netmaster.log
+			else
+				/contiv/bin/netmaster $debug -cluster-mode $plugin -listen-url $listen_url -control-url $control_url &>/var/contiv/log/netmaster.log
+			fi
+			echo "CRITICAL : Netmaster has exited. Trying to respawn in 5s"
 		fi
-		echo "CRITICAL: netmaster has exited, Respawn in 5"
 		sleep 5
 	done
 elif [ $netplugin == true ]; then
 	echo "Starting netplugin"
 	modprobe openvswitch
-	mkdir -p /var/contiv/log/
 
 	while true; do
-		if [ "$cstore" != "" ]; then
-			cstore_param="-cluster-store"
-		fi
-		if [ "$vtep_ip" != "" ]; then
-			vtep_ip_param="-vtep-ip"
-		fi
-		if [ "$vlan_if" != "" ]; then
-			vlan_if_param="-vlan-if"
+		if [ -f /tmp/restart_netplugin ]; then
+			if [ "$cstore" != "" ]; then
```

Comment:
Can you file an issue to clean this up and use the same startup script for both prod and test. Use docker cp to copy over the binary